__TESTS__/unit/analytics/analytics.node.test.ts

```diff
@@ -43,7 +43,7 @@ describe('Add analytics to a regular URL', () => {
         techVersion: '12.0.0',
         accessibility: true
       }
-    })).toContain('?_a=AZAlhAMD');
+    })).toContain('?_a=BAZAlhAMD');
```

Comment:
Shouldn't this be `BZAlhAMD` (only first letter changed from A to B?)

Reply:
Algorithm "B" also introduce a new letter fro product, which has a default of "A",
so as a result, "AZ..." will be "BAZ..."